backend/src/services/instance/instance.entity.ts

```diff
@@ -93,6 +100,7 @@ class Instance {
     createdBy,
     awsInstanceId,
     tenantId,
+    state,
```

Comment:
The problem with passing state like that is that state often changes on some domain event. For example: user create instance -> state='Creating'. Instance finished creating on AWS -> state='Active'. And so on... State is very different from name, for example, because name can be changed any time by user. However, state reflects status of entity.
So, I would do like that:
```
public static createNew({
  ...
}) {
  state = InstanceState.Created;
}
```
Trust me, when you will be asked to change some flow (add new step in instance creation/add new state) you will feel much better with such an explicit state definition. It is business requirements documentation for free in some degree